http-gateway/service/getCloudConfiguration_test.go

```diff
@@ -66,6 +67,8 @@ func TestRequestHandler_GetCloudConfiguration(t *testing.T) {
 			got.CloudCertificateAuthorities = ""
 			require.NotEqual(t, int64(0), got.CurrentTime)
 			got.CurrentTime = 0
+			require.NotEmpty(t, got.DeviceOnboardingCodeUrl)
+			expected.DeviceOnboardingCodeUrl = ""
```

Comment:
`expected.DeviceOnboardingCodeUrl` -> `got.DeviceOnboardingCodeUrl` ? Otherwise redundant because of line 22.

Reply:
Fixed